docs/getting-started/install/nightly-packages.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: base
+title: Nightly Snapshot packages
```

Comment:
Let's call it just `Nightly packages` to avoid confusion with the existing snapshot packages.